src/SFA.DAS.EAS.Portal.Client/Database/Models/AccountDocument.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Types;
+using System;
+
+namespace SFA.DAS.EAS.Portal.Client.Database.Models
+{
+    public class AccountDocument : Document
+    {
+        [JsonProperty("account")]
+        public Account Account { get ; set; }
+
+        [JsonProperty("accountId")]
+        public long AccountId => Account.Id;
```

Comment:
If we update the unique key to point to the new location of the account id, we shouldn't need this in `AccountDocument`.

Reply:
Feel free to make the change,

Reply:
Whether we can do this, depends on whether we have the luxury of being able to delete the existing database during the private beta (more details about this are in a different PR comment). If we can delete the database, I can make the changes required (they should be quick to make).